error: 'int64_t' was not declared in this scope

```
  [latte_int-1.7.6]   [spkg-install] In file included from /host/sage-musllinux_1_2_x86_64/include/zsolve/LinearSystem.hpp:30,
  [latte_int-1.7.6]   [spkg-install]                  from latte_4ti2_zsolve.h:28,
  [latte_int-1.7.6]   [spkg-install]                  from latte_4ti2_zsolve.cpp:23:
  [latte_int-1.7.6]   [spkg-install] /host/sage-musllinux_1_2_x86_64/include/zsolve/Integer.h:39:17: error: 'int64_t' was not declared in this scope
  [latte_int-1.7.6]   [spkg-install]    39 | inline int sgn (int64_t a)
  [latte_int-1.7.6]   [spkg-install]       |                 ^~~~~~~
  [latte_int-1.7.6]   [spkg-install] /host/sage-musllinux_1_2_x86_64/include/zsolve/Integer.h:33:1: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
  [latte_int-1.7.6]   [spkg-install]    32 | #include <sstream>
  [latte_int-1.7.6]   [spkg-install]   +++ |+#include <cstdint>
  [latte_int-1.7.6]   [spkg-install]    33 | 
```
https://github.com/passagemath/passagemath/actions/runs/16382249562/job/46303293569#step:12:26218

--- a/src/zsolve/Integer.h
+++ b/src/zsolve/Integer.h
@@ -23,6 +23,12 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #ifndef __4ti2_zsolve__Integer_
 #define __4ti2_zsolve__Integer_
 
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <inttypes.h>
+#endif
+
 #include "4ti2/4ti2_config.h"
 
 #ifdef _4ti2_HAVE_GMP

--- a/src/zsolve/zsolve.h
+++ b/src/zsolve/zsolve.h
@@ -23,11 +23,10 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #ifndef _4ti2_zsolve__zsolve_
 #define _4ti2_zsolve__zsolve_
 
+#ifdef __cplusplus
+#include <cstdint>
+#else
 #include <inttypes.h>
-
-#if 0
-typedef int int32_t;
-typedef long long int64_t;
 #endif
 
 #include "4ti2/4ti2_config.h"

